pyccl/cosmology.py

```diff
@@ -38,6 +39,8 @@ class MatterPowerSpectra(Enum):
     EMU = "emu"
     CAMB = "camb"
     CALCULATOR = "calculator"
+    BACCOEMU_NONLINEAR = "baccoemu_nonlinear"
+    EE2_NONLINEAR = "ee2_nonlinear"
```

Comment:
Can we be a bit more verbose and call this `EuclidEmu2` or something like that?

Reply:
Done